docs/guide_cog_creators.rst

```diff
@@ -0,0 +1,201 @@
+.. Cog Creator Applications
+
+.. role:: python(code)
+    :language: python
+
+================================
+Becoming an Approved Cog Creator
+================================
+
+This guide serves to explain the Cog Creator Application process and lays out the requirements to be a Cog Creator.
+
+----------------------------------
+Creating a Cog Creator Application
+----------------------------------
+
+.. note::
+  You will need to have created and published your cogs before you create a Cog Creator Application!
+  See `guide_cog_creation` and `guide_publish_cogs` for more information.
+
+Cog Creator Applications are hosted on the `cogboard <https://cogboard.red/c/apps/12>`__.
+To create an application, start a new topic in the "Applications" category and fill out all of the required information.
+QA reviews Cog Creator Applications for security and functionality on a first come, first serve basis.
+Once your application is reviewed, you will have 14 days to make any requested changes, or to check in with the member of QA who is reviewing your application.
+
+-----------------------------
+Requirements for Cog Creators
+-----------------------------
+
+The following is a list of the requirements for approved Cog Creators.
+QA uses this list to request changes for Cog Creator Applications.
+Handling these requirements before submitting your application can streamline the review process.
+Any Cog Creator that does not follow these requirements will have their repo removed from approved listings any may have their Cog Creator status revoked.
+
+- Readme that contains
+
+  - Repository Name
+  - Installation Instructions
+  - Setup Instructions (if applicable)
+  - Credits (if applicable)
+
+- Repo-wide info.json with the keys
```

Comment:
```suggestion
- Repo-wide ``info.json`` files with the keys
```

Reply:
Edited `files` -> `file` since there is only one repo `info.json` file